client_src/js/lib/modules/printerSettings.js

```diff
@@ -887,25 +890,27 @@ class PrinterSettings {
   static createServerResponseMessageForSaveAction(currentPrinterName, response) {
     let serverResponseMessage = `${currentPrinterName}: Settings Saved <br>`;
 
-    if (response.status.octofarm === 200) {
+    if (response) {
       serverResponseMessage += '<i class="fas fa-check-circle text-success"></i> OctoFarm<br>';
     } else {
       serverResponseMessage += '<i class="fas fa-exclamation-circle text-danger"></i> OctoFarm<br>';
     }
-    if (response.status.profile === 200) {
-      serverResponseMessage +=
-        '<i class="fas fa-check-circle text-success"></i> OctoPrint Profile<br>';
-    } else {
-      serverResponseMessage +=
-        '<i class="fas fa-exclamation-circle text-danger"></i> OctoPrint Profile<br>';
-    }
-    if (response.status.settings === 200) {
-      serverResponseMessage +=
-        '<i class="fas fa-check-circle text-success"></i> OctoPrint Settings <br>';
-    } else {
-      serverResponseMessage +=
-        '<i class="fas fa-exclamation-circle text-danger"></i> OctoPrint Profile <br>';
-    }
+
+    // TODO Different API
```

Comment:
Work in progress - core functionality I want to re-do. The problem is that the settings update is now meant only for network related changes. Printer connection settings or profile... do not belong in that settings update call.

So I would personally add separate save buttons for each set of settings and one button to "Apply All Changes"